app/screens/home/search/results/results.tsx

```diff
@@ -114,9 +123,45 @@ const Results = ({
         openGalleryAtIndex(galleryIdentifier, idx, items);
     }), [orderedFilesForGallery]);
 
-    const handleOptionsPress = useCallback(preventDoubleTap(() => {
-        // hook up in another PR
-    }), []);
+    useEffect(() => {
+        if (!lastViewedIndex) {
+            return;
+        }
+        const dismissBottom = async () => {
+            await dismissBottomSheet();
+        };
+        dismissBottom();
```

Comment:
Why is this needed? Can't we just call `dismissBottomSheet`?

Reply:
we discussed and handled this in our 1:1.